tests/xtimer_cli/main.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2018 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for xtimer API
+ *
+ * @author      Sebastian Meiling <s@mlng.net>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "shell.h"
+#include "xtimer.h"
+
+
+int cmd_xtimer_now(int argc, char **argv)
+{
+    (void)argv;
+    (void)argc;
+
+    uint32_t now = xtimer_now().ticks32;
+    printf("Success: xtimer_now(): [%"PRIu32"]\n", now);
+    return 0;
+}
+
+int cmd_get_metadata(int argc, char **argv)
+{
+    (void)argv;
+    (void)argc;
+
+    printf("Success: [%s, %s]\n", RIOT_BOARD, RIOT_APPLICATION);
+
+    return 0;
+}
+
```

Comment:
Didn't look it up, but is the `echo` cmd described in #10624 already used by riot_pal? If yes, maybe add it here?

Reply:
nope, no echo there, and I would rather make (as discussed offline) a PR introducing both (metadata and echo) as a shell module, and not start adding stuff here that is not needed. I need the metadata but I don't need or use `echo` right now, so why bother.